From 6fc44d1be25353353e6c1fcae9d98aea58c24eec Mon Sep 17 00:00:00 2001
From: Evangelos Foutras <evangelos@foutrelis.com>
Date: Thu, 8 Oct 2015 16:50:26 +0300
Subject: [PATCH] Use non-legacy edit-find icon instead of gtk-find

gtk-find is a legacy icon name found in the GNOME icon theme, but it's
not part of the Adwaita icon theme. gtk-find maps to edit-find via a
symlink so just use the latter.
---
 data/xfce4-appfinder.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/xfce4-appfinder.desktop.in b/data/xfce4-appfinder.desktop.in
index c4bd6ca..dc34d21 100644
--- a/data/xfce4-appfinder.desktop.in
+++ b/data/xfce4-appfinder.desktop.in
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
 Exec=xfce4-appfinder
-Icon=gtk-find
+Icon=edit-find
 StartupNotify=true
 Terminal=false
 Type=Application
-- 
2.6.1

